vcpp-client-reference/vcpkg.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": [
+    "grpc"
```

Comment:
we should specify versions here at some point. I dont really want to move from grpc 1.X to grpc 2.X automagically, which im pretty sure is what we're configuring here.

still it isnt a problem yet so this will do.

Reply:
Yes. Once we are sure this builds on the platforms we claim support for and address the other things, we'll commit to specific gRPC versions.